environment_agency_flood/env_agency_flood.py

```diff
@@ -151,7 +151,7 @@ def get_data(self) -> iter:
             endpoint = 'id/stations/{station_id}/readings.csv'.format(station_id=self.get_value(station,"items_stationReference"))
             
             params = dict(
-                date=self.date.strftime("%Y-%m-%d"),
```

Comment:
isoformat() is not working for me. It's probably versions collision. I am using Python 3.5.

Reply:
It should be in version 3.5
https://docs.python.org/3.5/library/datetime.html#datetime.date.isoformat